File: Java/백준/Bronze/2475. 검증수/검증수.java
import java.util.Scanner;

public class Main {
public static void main(String[] args) {

        Scanner s = new Scanner(System.in);

        int[] numbers = new int[5];
        double clarify = 0;
        for (int i = 0; i < 5; i++) {
            numbers[i] = s.nextInt();
            clarify += Math.pow(numbers[i], 2);
        }

        System.out.print((int) clarify % 10);
    }
}

File: C11/백준/Bronze/2475. 검증수/검증수.c
#include <stdio.h>
#include <math.h>

int main(void){
	
	int numbers[5];
	double clarify = 0;
	
	for(int i = 0; i < 5; i++){
		scanf("%d", &numbers[i]);
		clarify += pow(numbers[i], 2);
	}
	
	printf("%d", (int) clarify % 10);
	
	return 0;
}